discovery-provider/src/queries/get_tracks.py

```diff
@@ -35,6 +33,15 @@ def _get_tracks(session, args):
             TrackRoute, TrackRoute.track_id == Track.track_id
         ).filter(TrackRoute.slug == slug)
 
+    # Only return unlisted tracks if slug and user_id are present
+    if (
+        not "include_unlisted" in args
+        or not args.get("include_unlisted")
+        or not "slug" in args
```

Comment:
do we need the include_unlisted arg at all?

if not, for reasoning sake, I'd rewrite as

has_slug_and_id = "slug" in args and "user_id" in args

if not has_slug_and_id:
  filter(is_unlisted == False)

Reply:
How does GA handle unlisted tracks? I may come back to change this to allow "include_unlisted" to be false if included in the request as false so that unlisted tracks don't get picked up by search engines

Reply:
Went ahead and updated to not have that arg. We can add back if necessary but I think excluding it makes sense.